cypress/integration/chi-vue/data-table.spec.js

```diff
@@ -60,6 +60,10 @@ const DATA_TABLE_EVENTS = {
   EXPANSION: {
     EXPANDED: 'chiRowExpanded',
     COLLAPSED: 'chiRowCollapsed'
+  },
+  BULK_ACTIONS: {
```

Comment:
I'd name this BULK_ACTIONS_EVENTS

Reply:
I am using the same naming as the other events, e.g. TOOLBAR does not have TOOLBAR_EVENTS:

![image](https://user-images.githubusercontent.com/86958774/152759718-b76bc091-03f1-4df8-940c-beed91caa527.png)